### DISCUSSION TITLE
Support decimals in InputCount

### DISCUSSION THREAD

**Discussion by theorlovsky** [taiga-family/taiga-ui#3573] [Ideas]
While I understand the concept why "count" supports only integers, I find it really excessive creating an almost exact copy of this component just to be able to see the decimal part of the value. What do you think about adding this support to the existing component? Or any other thoughts?

**Comment by waterplea** [comment]
That would basically mean you want to add +/- buttons to `InputNumber`, right? Perhaps we can do that and just deprecate `InputCount` all together. After all, `InputNumber` + decimals="never"  is pretty much `InputCount` without those buttons.
